From c56c467f08469138001cc80a8de77e3f6a2aab8a Mon Sep 17 00:00:00 2001
Message-Id: <c56c467f08469138001cc80a8de77e3f6a2aab8a.1684964119.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1684964119.git.sharlatanus@gmail.com>
References: <cover.1684964119.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 16 May 2023 20:28:30 +0100
Subject: [PATCH v2 09/20] gnu: python-pyvo: Update to 1.4.1.

* gnu/packages/astronomy.scm (python-pyvo): Update to 1.4.1.
[build-system]: Switch to use pyproject-build-system.
[arguments]: Stop modifying standard phases.
[native-inputs]: Add python-setuptools-scm.
---
 gnu/packages/astronomy.scm | 19 ++++---------------
 1 file changed, 4 insertions(+), 15 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 9e3a36e1d1..29bc60ae98 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -1723,27 +1723,16 @@ (define-public python-poppy
 (define-public python-pyvo
   (package
     (name "python-pyvo")
-    (version "1.2.1")
+    (version "1.4.1")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "pyvo" version))
        (sha256
-        (base32 "1ri5yp6903386lkn79mdcmlax7zsfrrrjbcvb91wxydcc9yasc1n"))))
-    (build-system python-build-system)
-    (arguments
-     '(#:phases
-       (modify-phases %standard-phases
-         (replace 'check
-           (lambda* (#:key tests? #:allow-other-keys)
-             (when tests?
-               (invoke "python" "-m" "pytest" "--pyargs" "pyvo" "-k"
-                       (string-append   ; these tests use the network
-                        "not test_access_with_string"
-                        " and not test_access_with_list"
-                        " and not test_access_with_expansion"))))))))
+        (base32 "17acv1yhz1jrsx9f35nr1vg276ibaivh4i243qkmp6abzvfyg907"))))
+    (build-system pyproject-build-system)
     (native-inputs
-     (list python-pytest-astropy python-requests-mock))
+     (list python-pytest-astropy python-requests-mock python-setuptools-scm))
     (propagated-inputs
      (list python-astropy python-mimeparse python-pillow python-requests))
     (home-page "https://github.com/astropy/pyvo")
-- 
2.39.2

